apple-app-site-association.json

```diff
@@ -51,6 +52,7 @@
           "NOT *L3NpZ25f*",
           "NOT *L3VzZXJzL2F1dGgvZmFjZWJvb2svY2FsbGJhY2s=*",
           "NOT *L2lkZW50aXR5LXZlcmlmaWNhdGlv*",
+          "NOT *L3Jlc2V0X3Bhc3N3b3Jk*",
```

Comment:
this came after running yarn build

Reply:
Sailthru encodes links in their deep links so this will be the change-password link with Sailthru's encoding, although probably not sending any links to reset password

Reply:
Thanks for explaining 🙇‍♂️